sample_consensus/include/pcl/sample_consensus/sac.h

```diff
@@ -159,6 +161,17 @@ namespace pcl
       inline double 
       getProbability () const { return (probability_); }
 
+      /** \brief Set the number of threads to use or turn off parallelization.
+        * \param[in] nr_threads the number of hardware threads to use (0 sets the value automatically, a negative number turns parallelization off)
```

Comment:
Maybe explain the "default" value of threads and it's behavior (-1, 0, etc.)

Reply:
I think that's done in the parameter description? Or could you suggest an explanation/description?